src/app/(pages)/projects/[id]/page.tsx

```diff
@@ -0,0 +1,21 @@
+"use client";
+import { getProjectsById } from '@/app/Service/ProjectServices';
+import { Container, } from '@mui/material';
+import React from 'react';
+import { useQuery } from 'react-query';
+import { TabsComponent } from '@/app/components/tabs/Tabs';
+
+const IdProject = ({ params }: { params: { id: string } }) => {
+  //Get Project By ID 
+  const { data :ProjectsById, isLoading, error } = useQuery(
```

Comment:
why you fetch here and pass which fetched to tabs component, why not fetching from the first in tabs component?